R/antigen.garnish_predict.R

```diff
@@ -1810,8 +1809,43 @@ if (assemble & input_type == "peptide"){
 
     dt <- dt[pep_mut %like% "^[ARNDCQEGHILKMFPSTWYV]+$"]
 
+    if ("pep_wt" %chin% names(dt)){
+
+      message("Checking for non-standard AA one-letter codes in \"pep_wt\". Non-standard rows will be discarded.")
+
+      dt <- dt[pep_wt %like% "^[ARNDCQEGHILKMFPSTWYV]+$"]
+
+      if (nrow(dt) == 0) stop("All rows contained non-standard AAs in either pep_mut or pep_wt columns.")
```

Comment:
I would change this to `any`. In my opinion, if something is wrong about the input, the execution should not proceed, even if it is not just one line, because the output will not be as expected. A warning would be reasonable, but no on reads warnings.

Reply:
I'm fine with that. I will implement that change.  We will still need the AA whitelist elsewhere in the package for transcript level input because the GRCh/m38 db does have some peptides with non-standard AA afaik.